pkg/workspace/validate.go

```diff
@@ -48,3 +48,23 @@ func ValidateBindings(w []v1beta1.WorkspaceDeclaration, wb []v1beta1.WorkspaceBi
 	}
 	return nil
 }
+
+// ValidateOnlyOnePVCIsUsed checks that a list of WorkspaceBinding uses only
+// one persistent volume claim. This is useful to validate that WorkspaceBindings
+// are compatible with affinity rules enforced by the AffinityAssistant.
+func ValidateOnlyOnePVCIsUsed(wb []v1beta1.WorkspaceBinding) error {
```

Comment:
nit: maybe add to the comment that this condition (multiple pvcs) is valid for `WorkspaceBinding` in a PipelineRun, but not for a `TaskRun`

Reply:
good thinking; I've updated the comment to include that detail.

Reply:
>This is useful to validate that WorkspaceBindings
// in PipelineRuns are compatible with affinity rules enforced by the
// AffinityAssistant.

It's the other way around :) . We only call this function from `taskrun.go` because this condition is ok when declared in a `PipelineRun`

Reply:
Maybe 

// This is only useful to validate that WorkspaceBindings
// in TaskRuns are compatible with affinity rules enforced by the
// AffinityAssistant.

Reply:
oops, hah, thanks I've corrected it with your wording.